documentation/book/assembly-kafka-broker-configuration.adoc

```diff
@@ -10,10 +10,8 @@
 
 = Kafka broker configuration
 
-{ProductName} allows you to customize the configuration of Apache Kafka brokers.
-You can specify and configure most of the options listed in {ApacheKafkaBrokerConfig}.
-
-The only options which cannot be configured are those related to the following areas:
+{ProductName} allows you to customize the configuration of each Kafka broker in your Kafka cluster.
```

Comment:
_each Kafka broker_ suggests you can configure each Kafka broker differently. Which is not true. I think we should leave out _each_ or use something else.

Reply:
Thanks for the feedback. I was trying to imply that each Kafka broker is configured _separately_. I've changed to:

"...allows you to customize the configuration of the Kafka brokers in your Kafka cluster."